components/integrations/installed_oauth_apps/installed_oauth_apps.jsx

```diff
@@ -7,7 +7,7 @@ import {FormattedMessage} from 'react-intl';
 
 import {localizeMessage} from 'utils/utils.jsx';
 import BackstageList from 'components/backstage/components/backstage_list.jsx';
-import InstalledOAuthApp from '../installed_oauth_app';
+import InstalledOAuthApp from '../installed_oauth_app/index.ts';
```

Comment:
This can be reverted. Webpack knows to look for the index.ts when you import a folder.

Reply:
Oh yes, great catch :) thanks. I'll fix shortly